src/main/kotlin/no/nav/familie/ef/sak/opplysninger/søknad/SøknadService.kt

```diff
@@ -27,6 +31,24 @@ class SøknadService(
 
     private val logger = LoggerFactory.getLogger(javaClass)
 
+    fun hentSøknadsdatoer(behandlingId: UUID): SøknadDatoerDto? {
```

Comment:
Kan man bruke `hentSøknadsgrunnlag -> søknadsverdier` for å generere søknadsdatoer?